doc/scenario.md

```diff
@@ -0,0 +1,317 @@
+# Role Scenario
+
+The role uses a scenario of steps to determine what to do.
```

Comment:
The role uses a scenario to determine what to do.
Scenario consists of **steps** that can be configured.

(I don't like "scenario of steps" phrase, it's **role scenario** that consists of **steps**)